runtime/bindings/python/src/openvino/helpers.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+"""openvino helper functions."""
+
+
+from openvino import Function
+from openvino.inference_engine import IENetwork
+
+# TODO: will be removed after full updating MO with new api
```

Comment:
Add ticket number from this task here.

Reply:
@AlexeyLebedev1 you may add this ticket 69196 -- I think it'll be one scope of work there